### ISSUE TITLE
Hyperparameters for Training ShuffleNetV2+ Medium

### ISSUE THREAD

**Issue by iamhankai** [megvii-model/ShuffleNet-Series#10]
I have run the code of ShuffleNetV2+ Medium with default hyperparameters, and obtained the result: `Top-1 err = 0.248780,	Top-5 err = 0.077260`. 

The top-1 err **24.9** is a little bit different from the reported **24.3**. Could you provide the Hyperparameters for Training ShuffleNetV2+ Medium?

**Comment by nmaac** [comment]
Hi @iamhankai , the training setting for ShuffleNetV2+ Medium is the same with the default settings.

**Comment by iamhankai** [comment]
> Hi @iamhankai , the training setting for ShuffleNetV2+ Medium is the same with the default settings.

OK, thanks. I'll try it again.
